app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -262,12 +262,7 @@ class GithubWebhook extends RequestHandler<Body> {
         needsTests = !_allChangesAreCodeComments(file);
```

Comment:
Can the above be changed to `!_isATest(filename)`
```
          !filename.startsWith('dev/devicelab/bin/tasks') &&
          !filename.startsWith('dev/devicelab/lib/tasks') &&
```
?
At a minimum it those two lines could be removed.

Reply:
I think we want to check `_allChangesAreCodeComments`, because if the file isn't one of the above file types, but the only diffs are in comments, then that doesn't not make `needsTests`.

But you're right, I can remove the checks for the devicelab files, because even if that marks this `needs` test, we will certainly also mark the PR as having tests.